administrator/components/com_admin/sql/updates/mysql/4.0.0-2019-07-27.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE `#__banner_clients` ALTER `checked_out_time` DROP DEFAULT;
+ALTER TABLE `#__banner_clients` CHANGE COLUMN `checked_out_time` `checked_out_time` DATETIME NULL;
```

Comment:
```suggestion
ALTER TABLE `#__banner_clients` MODIFY COLUMN `checked_out_time` DATETIME NULL DEFAULT NULL;
```
1. CHANGE is only necessary when the column name shall be changed, too, otherwise we should use MODIFY.
This should be changed for all the following column change statements, too.
2. We should do that in 1 statement, so the database checker also can handle it. So change this line as shown here and remove the previous one with `DROP DEFAULT`. But I am not 100 % sure if that will work on 8.0, too, or if it needs the 2 separate staments. In other PRs, e.g. 25362 or 25360 it was done like I say, but if that fails then go back to separate statements and just change "CHANGE COLUMN `checked_out_time` `checked_out_time`" to "MODIFY COLUMN `checked_out_time`".